device/accton/x86_64-accton_as7116_54x-r0/Accton-AS7116-54X/port_config.ini

```diff
@@ -48,8 +48,8 @@ Ethernet45      77                Ethernet46/1      45
 Ethernet46      78                Ethernet47/1      46
 Ethernet47      79                Ethernet48/1      47
 Ethernet48      84,85,86,87       Ethernet49/1      48
-Ethernet52      80,81,82,83       Ethernet50/1      49
-Ethernet56      104,105,106,107   Ethernet51/1      50
-Ethernet60      108,109,110,111   Ethernet52/1      51
-Ethernet64      112,113,114,115   Ethernet53/1      52
-Ethernet68      116,117,118,119   Ethernet54/1      53
\ No newline at end of file
+Ethernet49      80,81,82,83       Ethernet50/1      49
```

Comment:
This does not seem to follow the convention where 2/4-lanes ports should reserve gaps in between.

Reply:
what gaps?